### DISCUSSION TITLE
Cockpit ideas that i would like to see.

### DISCUSSION THREAD

**Discussion by Di0so** [Drag0nD3str0yer/Moderately-Plane-Related#29] [Ideas]
Hello, your way of making cockpits are superb although not all of them are good, so here are cockpits that I would like to see, 

1 - F-5 cockpit, I believe you can make this cockpit, and if you really do please add the intake as well, or make it a separate part.
2 - F-84 cockpit, classic cockpit from the Korean war, and if you add it, consider making an intake part for it too.
3 - MiG 15/17 cockpit, one of my most favorite planes, if you could add it, just add a separate intake part for it too
4 - Su-27 cockpit, please do 
thats all

**Comment by yukikaze-zero** [comment]
Please do the two Seater F-104D cockpit
